hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/ICopyObject.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@ashishkumar50 , we usually leave an empty line before and after the link; see "Source File Headers for Code Developed at the ASF" in https://www.apache.org/legal/src-headers.html

BTW, the `<p>` tag is for javadoc.  That why we should remove it and leave an empty line.

Since it can pass the [rat test](https://github.com/apache/ozone/actions/runs/5027082009/jobs/9019199077?pr=4745#step:7:1), let's just merge this.